.github/workflows/publish.yml

```diff
@@ -0,0 +1,43 @@
+name: "WebGL SDK Publish"
+
+on:
+    push:
+        branches:
+            - main
```

Comment:
this should be updated to branch: dev

Reply:
Per your other question regarding push every release, this is now changed to trigger per release.